### PR TITLE
To Do List: Project 1 - List structure

### PR DESCRIPTION
**Pull request by Nedramrez**
- Set up a new project with webpack
- Add an index.html file as a template using the HTML Webpack Plugin.
- Add an HTML markup
- Add an empty To Do List placeholder 
- Add an index.html file set 
- Set an array of to-do tasks array of objects
- Set 3 keys for each object: description [string], completed [bool], index: [number]
- Write a function to populate an HTML list item element for each task
- Render the created list of tasks  dynamically in the dedicated placeholder
- The list appears in order of the index values for each task
- Create a style.css and set rules for the To-Do List 
- CSS loaded by Webpack Style/CSS Loader

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+![microverse icon](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This simple web page will be built using webpack and served by a webpack dev server.
+
+## Built With
+
+- JavaScript ES6
+- HTML
+- CSS
+
```
Comment (Ismailco) [inline README.md:12]: - [x] [OPTIONAL] I suggest adding a `Get started` section with instruction on how to clone the project and use npm to start development server or deployment for exemple

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+  </head>
+  <body>
+    <section class="to-do-list">
```
Comment (Ismailco) [inline src/index.html:9]: - [x] [OPTIONAL] Good work on the app layout, however I suggest centring the todo lidt to be more semantic and easy to work with

**Review by Ismailco — APPROVED** [review]
### Hi @Nedramrez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 2 above)
Comment (Nedramrez) [reply src/index.html:9]: Thank you @Ismailco very much! Done!

File: README.md (hunk 1 above)
Comment (Nedramrez) [reply README.md:12]: Thank you @Ismailco! I will do that when I will understand what and how to do it. :sweat_smile:

**Review by Davidosky007 — COMMENTED** [review]
Hi @Nedramrez 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the newly updated readme template, Kindly Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**